src/commands/poll/subcommands/create.ts

```diff
@@ -0,0 +1,113 @@
+import SubCommand from '../../../SubCommand';
+import CommandOption from '../../../CommandOption';
+import * as Discord from '../../../Discord';
+import { authorAvatarUrl } from '../../../cdn';
+import { EM_SPACE } from '../../../format';
+import ExecutionContext from '../../../ExecutionContext';
+
+const respondWithCountMismatch = (
+  ctx: ExecutionContext,
+  emojiCount: number,
+  descriptionCount: number,
+): Promise<void> => {
+  return ctx.interactionApi.respondWithError(
+    `You supplied ${emojiCount} emojis but ${descriptionCount} descriptions. ` +
+      `There should be exactly 1 description for each emoji, or no descriptions at all.`,
```

Comment:
Is there a reason these are two separate strings?
```suggestion
    `You supplied ${emojiCount} emojis but ${descriptionCount} descriptions. There should be exactly 1 description for each emoji, or no descriptions at all.`,
```

Reply:
To create a line break in between for code styling. The latter creates a new line 👌